gen2-cumulative-object-counting/trackable_object.py

```diff
@@ -0,0 +1,11 @@
+# from https://www.pyimagesearch.com/2018/08/13/opencv-people-counter/
+class TrackableObject:
```

Comment:
Could we move the declaration of this class to the `main.py`? It would be one file less and this class is compact anyways